.changeset/light-rectangles-rejoice.md

```diff
@@ -0,0 +1,70 @@
+---
+"@fluidframework/tree": minor
+---
+---
+section: tree
+---
+
+Allow associating metadata with Field Schema.
```

Comment:
```suggestion
Allow associating metadata with Field Schema
```
I think we've been not adding periods at the end of the description for previous changesets.